src/GoTrueClient.ts

```diff
@@ -1672,15 +1671,12 @@ export default class GoTrueClient {
   > {
     let codeChallenge: string | null = null
     let codeChallengeMethod: string | null = null
+
     if (this.flowType === 'pkce') {
-      const codeVerifier = generatePKCEVerifier()
-      await setItemAsync(
+      let { codeChallenge, codeChallengeMethod } = await getCodeChallengeAndMethod(
         this.storage,
-        `${this.storageKey}-code-verifier`,
```

Comment:
The `/PASSWORD_RECOVERY` suffix is needed [so that the PASSWORD_RECOVERY event is properly emitted](https://github.com/supabase/gotrue-js/pull/813)